python/cac_tripplanner/templates/home.html

```diff
@@ -230,7 +230,7 @@
 
 {%block jspage %}
 <script type="text/javascript">
-//jQuery.noConflict(); // because JotForms.
+jQuery.noConflict(); // because JotForms.
```

Comment:
This takes care of the home view, but the footer is used on other views (i.e. Learn list and detail), so they're getting console errors "prototype.forms.js:591 Uncaught TypeError: element.dispatchEvent is not a function" and "prototype.forms.js:575 Uncaught TypeError: element.attachEvent is not a function".